concordium-sdk/src/main/java/com/concordium/sdk/Client.java

```diff
@@ -153,6 +154,25 @@ public BlockSummary getBlockSummary(Hash blockHash) throws BlockNotFoundExceptio
         return blockSummary;
     }
 
+    /**
+     * Retrieves a {@link BlocksAtHeight}
+     * @param height the height to query blocks for.
+     * @return A {@link BlocksAtHeight} if one or more blocks was present at the given height.
+     * @throws BlockNotFoundException if no blocks were present at the given height.
+     */
+    public BlocksAtHeight getBlocksAtHeight(long height) throws BlockNotFoundException {
```

Comment:
I do not see here how you distinguish between the relative/absolute height.

https://github.com/Concordium/concordium-grpc-api/blob/89eb52e2e8f75e1337a2193cc3905ece68da6947/concordium_p2p_rpc.proto#L140

In the rust SDK I have the following input to the method https://github.com/Concordium/concordium-rust-sdk/blob/main/src/endpoints.rs#L89
(and a suitable From implementation to make it simpler to use).

Maybe in Java it would make more sense to just have two different methods, one for relative one for absolute query.

Reply:
You are correct. I choose to keep it at one method - my proposal resembles the rust API a bit.